app/src/ui/lib/toggletipped-content.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react'
+import { ITooltipProps, Tooltip } from './tooltip'
+import { createObservableRef } from './observable-ref'
+import classNames from 'classnames'
+import { AriaLiveContainer } from '../accessibility/aria-live-container'
+
+/**
+ * IToggledtippedContentProps is a superset of ITooltipProps but does not
+ * define the `target` prop as that's set programatically in render
+ */
+interface IToggledtippedContentProps
+  extends Omit<ITooltipProps<HTMLElement>, 'target'> {
+  /** The tooltip contents */
+  readonly tooltip: JSX.Element | string | undefined
+
+  /**
+   * An optional additional class name to set on the tooltip in order to be able
+   * to apply specific styles to the tooltip
+   */
+  readonly tooltipClassName?: string
+
+  /** An optional class name to set on the wrapper element */
+  readonly className?: string
+
+  /** An optional aria-label property in case children is not descriptive
+   * - for example an icon */
+  readonly ariaLabel?: string
+}
+
+/**
+ * A less keyboard toggleable version of the Tooltip content component for when
+ * it's acceptable to add a wrapping element around the content. The content
+ * will be wrapped in a button as it is toggleable. supports all the options
+ * that the Tooltip component does without having to worry about refs.
+ **/
+export class ToggledtippedContent extends React.Component<IToggledtippedContentProps> {
+  private buttonRef = createObservableRef<HTMLButtonElement>()
+  private shouldForceAriaLiveMessage = false
+  private tooltipVisible = false
+
+  private onToggle = () => {
+    this.shouldForceAriaLiveMessage = !this.shouldForceAriaLiveMessage
+  }
+
+  private onTooltipVisible = () => {
+    this.tooltipVisible = true
+  }
+
+  private onTooltipHidden = () => {
+    this.tooltipVisible = false
+  }
+
+  public componentDidMount(): void {
+    this.buttonRef.subscribe(this.onButtonRef)
+  }
+
+  private onButtonRef = (ref: HTMLButtonElement | null) => {
+    if (ref === null) {
+      this.buttonRef.current?.removeEventListener(
```

Comment:
It doesn't look like we're ever subscribing to `tooltip-shown` so I don't understand why we're attempting to remove it here. Similarly it appears we never unsubscribe from the `tooltip-hidden` event?

Reply:
Hmm.. This looks messed up  from what I thought I landed on.... and am thinking.. how did this even work last time I tested it.  🤦 Sorry for definitely weird code.

Reply:
Here is what I had before and have updated too.. I am sorry for wasting your time.
![CleanShot 2023-04-05 at 10 22 02](https://user-images.githubusercontent.com/75402236/230110494-0bdf561a-02b9-4e3b-a32a-d37ca39ce235.png)